code/modules/mob/living/carbon/xenomorph/say.dm

```diff
@@ -19,7 +19,7 @@
 	return "</span>"
 
 /mob/living/carbon/xenomorph/hivemind/hivemind_start()
-	return "<span class='game say hivemind xenohivemind'>The Hivemind"
+	return "<span class='game say hivemind xenohivemind'><span class='name'>The Hivemind</span>"
```

Comment:
Looks like Psykzz's hivemind bolding commits snuck in here somehow, you'll have to remove those

Reply:
yeah I haven't used git before, I cherry picked the one commit to another branch and made a new pull request